### ISSUE TITLE
Smoother sign-up experience

### ISSUE THREAD

**Issue by rajadain** [WikiWatershed/model-my-watershed#399]
Currently, after signing up, one receives an activation link in the email. Clicking the link takes one a to a page with a message like this:

**Activation Successful**
If you had work in progress, please switch back to that tab and login to save your work.

This is a rather clunky approach. We should think about ways to save the project state and recreate it after they activate their link in that tab, rather than instructing them to go back to some other tab.

**Comment by rajadain** [comment]
More discussion here: https://github.com/WikiWatershed/model-my-watershed/pull/385/files#r32458205

![image](https://cloud.githubusercontent.com/assets/1430060/8169440/9b67b530-1377-11e5-980a-6b9f11454242.png)

**Comment by mmcfarland** [comment]
We had a long conversation about dealing with in-progress registrations.  There are a lot of edge cases with saving the "current" project that would make this tricky.  I like the simple approach of "create an account, then login where you've created your work and that will seamlessly be saved" as opposed to managing extra state for potential use, figuring out which of that state is appropriate to restoring (multiple in progress projects), clearing out state, and so forth.  It's straightforward for emails with links to open new tabs.  What if I'm on a desktop using the app, register and then use my phone to activate the account via email there?  I'd prefer it not to recreate a session there, when I have a session open in my desktop tab.  Introducing bi-directional communication to the app page to keep it in sync with other browser/browser tab actions also increases the complexity.

I think there are some shortcuts we can take to get an improved workflow without tons of extra complexity, but I'm for keeping this low priority for now.

**Comment by rajadain** [comment]
Agreed. I just added it so its out there and we have a place to record these discussions.
